src/base/static/client/vector-tile-client.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  fetchStyle: async styleUrl => {
+    const response = await window.fetch(styleUrl);
```

Comment:
nit: for future reference, we may want to consider normalizing on a more robust client instead of the global `fetch`.

I've used this one before, and it's pretty good: https://www.npmjs.com/package/superagent

But as long as we have our network requests abstracted into respective api clients, it should be easy enough to refactor it later.

Reply:
Cool-- I think I'll leave `fetch` in place for now, but I'll look into other HTTP clients for future refactoring.